.travis.yml

```diff
@@ -1,54 +1,37 @@
 # @file
-# .travis.yml - Drupal 8 Travis CI Integration
+# Travis CI integration.
 
-language: php
+git:
+  depth: 5
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
 
-sudo: false
+language: php
 
 php:
   - 5.6
   - 7
 
-# Set up environment variables.
 env:
   global:
-    # Global
-    - PATH="$PATH:$HOME/.composer/vendor/bin"
-
-    # Composer
+    # Ensure composer is able to apply patches.
     - COMPOSER_EXIT_ON_PATCH_FAILURE=1
-
-    # DRUPAL_TI Config.
-    - DRUPAL_TI_MODULE_NAME="decoupled_auth"
-    - DRUPAL_TI_SIMPLETEST_GROUP="decoupled_auth"
-
-    # DRUPAL_TI Env
-    - DRUPAL_TI_ENVIRONMENT="drupal-8"
-    - DRUPAL_TI_CORE_BRANCH="8.2.x"
-
-
-    # Define runners and environment vars to include before and after the
-    # main runners / environment vars.
-    - DRUPAL_TI_SCRIPT_DIR_BEFORE="./travis-ci/before"
-    - DRUPAL_TI_SCRIPT_DIR_AFTER="./travis-ci/after"
-
     # DRUPAL_TI Testing.
-    - DRUPAL_TI_DB="drupal_travis_db"
-    - DRUPAL_TI_DB_URL="mysql://root:@127.0.0.1/drupal_travis_db"
-    - DRUPAL_TI_WEBSERVER_URL="http://127.0.0.1"
-    - DRUPAL_TI_WEBSERVER_PORT="8080"
-    - DRUPAL_TI_SIMPLETEST_ARGS="--verbose --color --concurrency 4 --url $DRUPAL_TI_WEBSERVER_URL:$DRUPAL_TI_WEBSERVER_PORT"
-    - DRUPAL_TI_PHPUNIT_ARGS=""
-    - SIMPLETEST_BASE_URL="http://127.0.0.1:8080"
-
-    # Debug options
-    #- DRUPAL_TI_DEBUG="-x -v"
-    #- DRUPAL_TI_DEBUG_FILE_OUTPUT="all"
+    - SIMPLETEST_DB="mysql://root:@localhost/drupal_travis_db"
+    - SIMPLETEST_BASE_URL="http://localhost:8888"
 
   matrix:
-    # @todo Move webtests to phpunit tests, then remove simpletest runner.
-    - DRUPAL_TI_RUNNERS="phpunit-core"
-    - DRUPAL_TI_RUNNERS="simpletest"
+    # Latest stable release
+    - DRUPAL_CORE="~8.0@stable"
+    # Latest dev branch
+    - DRUPAL_CORE="8.4.x-dev"
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - env DRUPAL_CORE="~8.0@dev"
```

Comment:
This should be matching the dev branch, so `- env DRUPAL_CORE="8.4.x-dev". Might need to fix this on contacts also...